src/plugins/RendererPlugin/components/Renderer.tsx

```diff
@@ -33,7 +33,7 @@ export const Renderer: React.SFC<{
   return (
     <>
       {components.map((Component: any) => (
```

Comment:
**Change (Blocking)**: Do not use any. 
**Suggestion**: Use better variable name such as `componentsDataArray`.
**Suggestion** Take component inside the map in a variable and then use that variable